pylibui/controls/button.py

```diff
@@ -22,7 +22,8 @@ def handler(window, data):
             self.onClick(data)
             return 0
 
-        self.clickHandler = libui.uiButtonOnClicked(self.control, handler, None)
+        self.clickHandler = libui.uiButtonOnClicked(self.control, handler,
+                                                    None)
```

Comment:
On github it doesn't show any 80 char delimiter, so it was easier for me to ask.. :)